docs/api-testing/composer.md

```diff
@@ -2,82 +2,601 @@
+
+2. Add a `Set (variable)` component by entering/selecting the following in the Composer:
+    * Var (the variable name) - `access_token`
+    * Variable mode (the variable type) - `String`
+    * Value - `${authPayload.access_token}`
+
+    <img src={useBaseUrl('/img/api-testing/int-assign-token.png')} alt="Setting the variable"/>
+
+  This step takes the `access_token` variable in the `authPayload` response, and sets it as `access_token`; the response body from the original post call was saved to a variable called `authPayload`. The access key for the token is `access_token`, which can be found by calling `authPayload.access_token`.
+
+  :::note
+  The dollar sign and brackets are necessary when referencing variables so that Sauce Labs API Testing knows to interpret what’s between the brackets instead of using it literally.
+  :::
+
+  Variables are used to store data temporarily for a test, but you can use the Sauce Labs API Testing Vault for permanent variables. For more information, see [Creating Reusable Variables and Snippets with the Vault](/api-testing/vault)).
+
+3. Make follow-up calls.
+
+  In the following example, the API has a cart function that requires a user token to add items to a cart or view items currently in the cart. Use a `PUT` request to the cart endpoint to update the cart. Use the access token granted by the authentication server to add items to a cart by setting the `token` header to `${access_token}`.
+
+  <img src={useBaseUrl('/img/api-testing/int-set-token-header.png')} alt="Setting the token header"/>
+
+  You can also reuse access tokens:
+
+  <img src={useBaseUrl('/img/api-testing/int-reuse-tokens.png')} alt="Reusing tokens" width="600"/>
+
+###  Test Interactions between Endpoints
+
+In the following example, there is an API endpoint that produces an array of all the available products and another endpoint that shows the details of a specific product based on its ID.
+
+  ```http request
+  http://demoapi.apifortress.com/api/retail/product
+  http://demoapi.apifortress.com/api/retail/product/${id}
+  ```
+
+To create an integration test to test the interaction between the endpoints:
+
+  1. Call the product listing endpoint and assign the response to the `productsPayload` variable.
+
+  2. Add a `for each` assertion and reference the `productsPayload.products` object.
+
+  :::note
+  In a scenario in which the response contains many products, it may be useful to pick a few at random by using `pick(n)`.
+  :::
+
+  3. Test the response payload for the endpoint.
+
+  4. Add a new `Set (variable)` assertion to set the `id` variable as every single `productsPayload.product` that is returned. In the following example, the string is `${_1.id}`. The system uses `_1` automatically when recognizing a subroutine, which makes it easier when there are multiple sub-levels.
+
+    <img src={useBaseUrl('/img/api-testing/int-test-endpoints.png')} alt="Testing interactions between endpoints" width="600"/>
```

Comment:
@LoryApiFortress I originally replaced it with a dummy url because yours includes "apifortress," which will be removed as part of rebranding. In addition, screenshots are meant to orient the user, not give them values to enter as they work through steps. If the user is meant to enter a value, including it in a screenshot is a bad way to do that because they can't select and copy anything.

However, I removed my dummy url from other screenshots and reverted to the apifortress url because making all the screenshots consistent would take too much time. I must have missed this one, and I will fix it. At some point, after the UI is rebranded, I **will** have to go back through all the docs and update the screenshots.